src/main/java/campus/data.txt

```diff
@@ -0,0 +1,3 @@
+T | 0 | do homework
+E | 0 | dinner with friends | 1900 29/01/2024 | 2200 29/01/2024
+D | 0 | buy cny decorations | 2359 30/01/2024
```

Comment:
I'm not sure if you're supposed to include the data file into the repository as source code. Consider using gitignore.